guides/doc-Planning_Guide/topics/Glossary.adoc

```diff
@@ -156,7 +158,11 @@ To see the full list of services, enter the `{foreman-maintain} service list` co
 
 [[varl-Glossary_of_Terms-Foreman_Hooks]]
 *Foreman Hook*:: An executable that is automatically triggered when an orchestration event occurs, such as when a host is created or when provisioning of a host has completed.
-
+ifdef::satellite[]
++
+Note that Foreman Hook functionality is deprecated and will be removed in the next {Project} version.\
++
```

Comment:
```suggestion
```

Reply:
We should state that webhooks is an alternative now, @ares ?

Reply:
Fixed